### PR TITLE
ERT Update

### PR DESCRIPTION
**Pull request by SamCroswell**
Been meaning to do this for a while.

ERT is no longer instantly deployed upon dual swipe. A reason is entered in the card swiper, a swipe is made, then CentComm gets the message. They can, then, deploy the ERT if it is deemed neccisary.

uisnap2
message

Upon hitting 'Reply' the admins get the choice to either deploy the ERT or decline the request with a message.

Re-maps the specops area of CentComm (Unfortunately, these are not mapmerged due to an error on my part when operating the tool. Noone is working on Z2 currently anyhow, so it should not be a big deal. If any maintainers take issue with it, please notify me.)

specopsv2

Changes how ERT works. There are now 3 types of ERT:

 - Code Amber ERT

The lowest tier of ERT, all members of the team (Aside from Engineer) are in armored ERT vests instead of their RIG suits, and are given basic gear to complete their task. In the event there are no admins online when an ERT request is made, this one will be automatically deployed.

- Code Red ERT

The middle tier of ERT, all squad members are given more advanced gear to get the job done (Including their EVA suits). Only to be dispatched in severe emergencies, this team has formidable gear that the station would have a hard time matching on its own.

- Code Gamma ERT

An elite ERT sent when the deathsquad is not appropriate, but their level of firepower is. All members of this team are given some variation of pulse weaponry and advanced gear the station would not ever be able to get on their own.

Lastly, this also does away with ERT types. Instead, to encourage a balanced ERT loadout, there is a maximum of 3 of any ERT member type (Aside from Commander) per team. This encourages team diversity, and allows people to play roles they enjoy without flooding the entire team with security officers.

### CODE REVIEW

**Comment by tigercat2000** [comment]
That nanoUI looks a bit odd. If I understand correctly, it says
`Select ERT type` followed by the button to input the reason they are being called.

Another note. Something that may be a good idea- transfer the ERT reason to any spawned members via notes if an admin approves it, or if it's an auto-ERT, could potentially be helpful.

**Comment by SamCroswell** [comment]
That was patched. The screenshot was taken before it was fixed. Currently it says 'Reason for ERT Call:'

**Comment by Fox-McCloud** [comment]
Definitely agree with this. Given how players treat ERT as a "get us out of anything" card, there needs to be appropriate limitations on it. Right now there's never a reason to NOT call an ERT at the first sign of trouble (for anything) for literally anything--you end up with a bunch of heavily armed, heavily armored staff patrolling the halls.

Seems like a well thought out approach that cuts down on this behavior and has appropriate safeguards if admins happen to not be online

:+1:  from me.

**Comment by FalseIncarnate** [comment]
Is team size still tied to server population?

Also, is the reason visible to sign-ups WHILE they select their role?

Assuming both are true, :+1:

**Comment by Aurorablade** [comment]
please please do not take away janitorial ert! *makes sad puppy eyes*

**Comment by Spacemanspark** [comment]
Hrm...
When you went on a rant yesterday about how much you hated ERT worked, you didn't mention anything about the amber alert idea.
That alone lets me give me a :+1: to this, as, in the case admins aren't online, an ERT team that won't completely fuck over rev rounds (but at least gives the crew some assistance.) can be sent.

**Comment by SamCroswell** [comment]
My issue with the janitorial ERT team is that I'm afraid it would be used as a joke by most people, and then one of the six ERT slots is taken up by a dude with nothing to fight the xenos, repair breaches, or remove borers with but a mop.

A busted station is an emergency, a station overrun with hostile forces is an emergency, a station full of cripples with no doctors is an emergency.

The station being dirty just doesn't strike me as an emergency, and I don't see it as worth the uselessness that will come of a team of the times janitor is taken but not needed.

Basically: I think Janitor adds clutter, but not much else, to the ERT system.

**Comment by FalseIncarnate** [comment]
Hmm, maybe that was illusion created by disinterested players not joining on lower pop rounds.

As for janitorial ert: they largely get called for horrible messes like biomass, vines, and admin jokes. Their telescythes are fairly powerful though, having NOSHIELD, edge/sharp, and the ability to be collapsed. Total removal would be less than ideal, but perhaps limit them to an amber-only role? This way they exist, but won't fill slots for more dangerous missions.

**Comment by Aurorablade** [comment]
I agree with amber alert only for janitorial, would make sense.

**Comment by CrAzYPiLoT-SS13** [comment]
Same here

2015-07-21 13:21 GMT+02:00 Aurorablade <notifications@github.com>:

**Comment by KasparoVy** [comment]
Sounds pretty great!

Gamma alert janitor ERT anyone?

**Comment by TheDZD** [comment]
Just going to mark this DNM so nobody accidentally merges it. Just tell one of us when it's ready.

**Comment by SamCroswell** [comment]
Due to how ERT is currently set up (The respond button being a quick link to the admin command to dispatch an ERT instead of being linked to the keycard swiper in any way) it would either have to be poorly coded or a roundabout pain in the butt to gett the ERT to have the station's reason for calling that ERT in particular displayed to them.

Good to merge whenever, presuming noone has any arguments.

**Comment by Aurorablade** [comment]
RIP Janitorial ERT :(
